### ISSUE TITLE
Is it possible, when selecting "Unit of measure: Solids/t", to reduce the value to a comma?

### ISSUE THREAD

**Issue by vitalii2011** [jtsage/FS22_ProductionInspector#23]
Or take the recalculated values from https://github.com/VertexFloat/additionalUnits
![fsScreen_2024_04_16](https://github.com/jtsage/FS22_ProductionInspector/assets/41552153/bb9f7b1b-c333-4d49-bc02-32b0148af6da)
![fsScreen_2024_04_16 (2)](https://github.com/jtsage/FS22_ProductionInspector/assets/41552153/e03c4424-77a5-48aa-a903-156ca739a5b8)

**Comment by jtsage** [comment]
Yeah, I really need to get back to this mod and do some cleanup and fixes.  Alas, I don't personally use it much anymore (I do mostly MP, and this one hasn't made the cut with our voting process)

I do have control over the precision of the numbers, but what is that measurement in the screen capture?  metric ton? (curiously called shortTon in the giants l10n stuff - which it is not. A true short ton is 2,000 imperial pounds, which it appears is what I used for "imperialTon" (I think the bit of research I did led me to believe that imperial long tons (2,240 imperial pounds) is *very* rarely used now, and only in pretty specific instances)

[self.unitsToIndex.MT]      = { precision = 3, isWeight = true,  text = "unit_tonsShort",    factor = 0.001 },
[self.unitsToIndex.T]       = { precision = 3, isWeight = true,  text = "unit_fsgUnitConvert_imperialTon",     factor = 0.0011023 },
